File: app/src/main/java/com/brainwellnessspa/ReferralModule/Model/AllContactListModel.java
package com.brainwellnessspa.ReferralModule.Model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class AllContactListModel {
    @SerializedName("ResponseData")
    @Expose
    private ResponseData responseData;
    @SerializedName("ResponseCode")
    @Expose
    private String responseCode;
    @SerializedName("ResponseMessage")
    @Expose
    private String responseMessage;
    @SerializedName("ResponseStatus")
    @Expose
    private String responseStatus;

    public ResponseData getResponseData() {
        return responseData;
    }

    public void setResponseData(ResponseData responseData) {
        this.responseData = responseData;
    }

    public String getResponseCode() {
        return responseCode;
    }

    public void setResponseCode(String responseCode) {
        this.responseCode = responseCode;
    }

    public String getResponseMessage() {
        return responseMessage;
    }

    public void setResponseMessage(String responseMessage) {
        this.responseMessage = responseMessage;
    }

    public String getResponseStatus() {
        return responseStatus;
    }

    public void setResponseStatus(String responseStatus) {
        this.responseStatus = responseStatus;
    }

    public class ResponseData {

        @SerializedName("Isinsert")
        @Expose
        private String isinsert;

        public String getIsinsert() {
            return isinsert;
        }

        public void setIsinsert(String isinsert) {
            this.isinsert = isinsert;
        }
    }
}

File: app/src/main/java/com/brainwellnessspa/referralModule/models/AllContactListModel.kt
package com.brainwellnessspa.referralModule.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AllContactListModel {
    @SerializedName("ResponseData")
    @Expose
    var responseData: ResponseData? = null

    @SerializedName("ResponseCode")
    @Expose
    var responseCode: String? = ""

    @SerializedName("ResponseMessage")
    @Expose
    var responseMessage: String? = ""

    @SerializedName("ResponseStatus")
    @Expose
    var responseStatus: String? = ""

    inner class ResponseData {
        @SerializedName("Isinsert")
        @Expose
        var isinsert: String? = ""
    }
}